temporal_sqlalchemy/bases.py

```diff
@@ -21,9 +23,9 @@
 
 
 class EntityClock(object):
-    tick = sa.Column(sa.Integer, primary_key=True, autoincrement=False)
-    timestamp = sa.Column(sa.DateTime(True),
-                          server_default=sa.func.current_timestamp())
+    id = sa.Column(sap.UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
+    tick = sa.Column(sa.Integer, nullable=False)
+    timestamp = sa.Column(sa.DateTime(True), server_default=sa.func.current_timestamp())
```

Comment:
The `timestamp` column wasn't changed in this PR.  I only fixed a code indent.

Adding `nullable=False` here wouldn't add much value since it has a server generated default. The `entity_id` columns on the history tables are also missing a `NOT NULL` constraint.  So if you strongly believe it should be fixed, it should go into a separate PR as this one is just to remove the compound primary keys.